Products/CMFPlone/skins/plone_scripts/getObjSize.py

```diff
@@ -35,7 +35,7 @@
 if not size:
     return '0 %s' % smaller
 
-if same_type(size, 0) or same_type(size, 0L):
+if same_type(size, 0) or same_type(size, 0):
```

Comment:
This now says the same thing twice.
And on Python 2 it no longer recognizes longs.
`isinstance six.integertypes` or whatever it is called is probably better.

Reply:
done